backend/danswer/connectors/models.py

```diff
@@ -20,3 +22,21 @@ class Document:
 
 def get_raw_document_text(document: Document) -> str:
     return "\n\n".join([section.text for section in document.sections])
+
+
+class InputType(str, Enum):
+    PULL = "pull"
+    FILE = "file"
```

Comment:
what is file? What happened to the push, pull, index all type of flows? do we need to discuss this?